uwblueprint/project-read-frontend
# Export Button

### Notion ticket link



[Data export](https://www.notion.so/uwblueprintexecs/Data-export-7d5d9331d70d4bfaa25e5187617ec4d4)



### Implementation description

- Create a button that zips together the CSV responses from endpoints implemented in the backend.
- Created new export API calls



### Steps to test

1. Press the "Export data" button and it should download a zip file with the CSVs
